recipes/lastz/meta.yaml

```diff
@@ -1,13 +1,13 @@
 package:
   name: lastz
-  version: "1.0.4"
+  version: "1.04.15"
```

Comment:
you could use the jinja templates here and reuse that in your recipe

Reply:
OK